components/ProfileMenuMemberships.js

```diff
@@ -57,7 +57,7 @@ const MembershipLine = ({ user, membership }) => {
           href={getSettingsRoute(membership.collective, null, user)}
           ml={1}
           color="black.500"
-          title={intl.formatMessage({ id: 'AdminPanel.button', defaultMessage: 'Admin' })}
+          title={intl.formatMessage({ id: 'Member.Role.ADMIN', defaultMessage: 'Admin' })}
```

Comment:
Having separate strings for "Admin" was done on purpose. The fact that they're the same word is an English specificity as they're in fact different concepts, let's take FR as an example:
- Admin (role): Administrateur
- Admin (button, like in "Admin panel"): Admin / Administration

Reply:
I recommend adding the string to `DUPLICATED_IGNORED_IDS` to get rid of the warning

Reply:
I recommend adding the string to `DUPLICATED_IGNORED_IDS` to get rid of the warning

Reply:
Cool. I effectively had a doubt and this is why I wanted you to check.